### PR TITLE
feat: Sync with Crowdin

### PR DESCRIPTION
**Pull request by goodhoko**
@tsusanka @vdovhanych As agreed on today's standup we'd like to include this in the upcoming release.

### CODE REVIEW

File: packages/suite-data/files/translations/en.json [hunk 1]
```diff
@@ -630,9 +640,9 @@
 	"TR_EXCHANGE_PROVIDER": "Provider",
 	"TR_EXCHANGE_RATE": "Price",
 	"TR_EXCHANGE_RECEIVE_ACCOUNT_QUESTION_TOOLTIP": "This is the account within your wallet where you'll find your coins once the transaction is finished.",
-	"TR_EXCHANGE_RECEIVE_NON_SUITE_ACCOUNT_QUESTION_TOOLTIP": "Receiving account is outside of Suite.",
 	"TR_EXCHANGE_RECEIVE_ADDRESS_QUESTION_TOOLTIP": "This is the specific alphanumeric address that will receive your coins. Verify this address on your Trezor.",
-	"TR_EXCHANGE_RECEIVE_NON_SUITE_ADDRESS_QUESTION_TOOLTIP": "This is the specific alphanumeric address that will receive your coins.",
+	"TR_EXCHANGE_RECEIVE_NON_SUITE_ACCOUNT_QUESTION_TOOLTIP": "TR_EXCHANGE_RECEIVE_NON_SUITE_ACCOUNT_QUESTION_TOOLTIP",
+	"TR_EXCHANGE_RECEIVE_NON_SUITE_ADDRESS_QUESTION_TOOLTIP": "TR_EXCHANGE_RECEIVE_NON_SUITE_ADDRESS_QUESTION_TOOLTIP",
```
Comment (goodhoko) [inline packages/suite-data/files/translations/en.json]: @LukasRada These were added by you [here](https://github.com/trezor/trezor-suite/commit/5cf184412289711766c25ed43c3f6f8fce448968). Can we publish this as is or shall we rather wait for (quick) fix?

Comment (LukasRada) [reply packages/suite-data/files/translations/en.json]: Please wait for fix. To what branch I'll create PR? feat/crowdin-sync or develop?

Comment (goodhoko) [reply packages/suite-data/files/translations/en.json]: Can you maybe commit (and push) directly to this branch? I'll then reorder the commits and run the sync again.

Comment (LukasRada) [reply packages/suite-data/files/translations/en.json]: Done.

Comment (goodhoko) [reply packages/suite-data/files/translations/en.json]: Thanks. I force pushed the commits now in reverse order and with your new translations in the localization file.

Resolving.,

**Comment by goodhoko** [comment]
@tsusanka @vdovhanych Feel free to merge this when it suits you.

There's one additional commit by @LukasRada that adds some forgotten translations for Invity features we discovered looking at the diff. (Good we decided to do this within this release. 👐 )

**Review by tsusanka — APPROVED** [review]

**Pull request merged**